src/Competence/Competence.styled.js

```diff
@@ -0,0 +1,217 @@
+import styled from "styled-components";
+
+export const CompetenceSection = styled.section``;
+
+export const CompetenceContainer = styled.div`
+  width: 100%;
+  max-width: 1366px;
+  margin: 0 auto;
+
+  @media (max-width: 991px) {
+    max-width: max-content;
+  }
+`;
+
+export const CompetenceWrapper = styled.div`
+  margin: 0 19px 20px;
+  padding: 33px 120px 0 118px;
+
+  @media (max-width: 991px) {
+    margin: 0;
+    padding: 0 10px;
+  }
+`;
+
+export const CompetenceInnerWrapper = styled.div`
+  display: flex;
+  gap: 70px;
+
+  @media (max-width: 991px) {
+    flex-direction: column;
+    gap: 0;
+  }
+`;
+
+export const CompetenceLeftBox = styled.div`
+  width: 100%;
+  max-width: 50%;
+`;
+
+export const CompetenceRightBox = styled.div`
+  width: 100%;
+  max-width: 50%;
+
+  @media (max-width: 991px) {
+    max-width: 262px;
+  }
+`;
+
+export const CompetenceTitle = styled.div`
+  width: 100%;
+  max-width: 406px;
+  position: relative;
+  margin: 3px 0px 52px 0px;
+  font-family: "Nunito-bold";
```

Comment:
font-family is not applying i.e naming conventation

Reply:
done